lib/index.js

```diff
@@ -3,26 +3,23 @@
  *
  * This modules exposes an API for accessing a redis key-value cache.
  *
- * Copyright (c) 2012-2018 Digital Bazaar, Inc. All rights reserved.
+ * Copyright (c) 2012-2022 Digital Bazaar, Inc. All rights reserved.
  */
-'use strict';
+import * as bedrock from '@bedrock/core';
+import {Client} from './client.js';
+import './config.js';
 
-const bedrock = require('bedrock');
-const Client = require('./client');
-
-// module API
-const api = {};
-module.exports = api;
-
-api.client = null;
-api.Client = Client;
-
-// load config
-require('./config');
-
-bedrock.events.on('bedrock.start', () =>
-  bedrock.events.emit('bedrock-redis.ready'));
-
-bedrock.events.on('bedrock.init', () => {
-  api.client = new Client().client;
+export let client = null;
+bedrock.events.on('bedrock.start',
+  () => bedrock.events.emit('bedrock-redis.ready')
+);
+bedrock.events.on('bedrock.init', async () => {
+  const clientInstance = new Client();
+  clientInstance.connect();
+  client = clientInstance.client;
```

Comment:
Any reason not to do this?

```suggestion
  client = new Client();
  client.connect();
```

Reply:
Yeah, the problem is `client = new Client()` is an instance of the `Client` class implemented by this module, not the node-redis client. I tried to keep the structure that same as it was before, but I'm struggling to see the benefits of including this `Client` class at all. It seems like it would be simpler to just create the node-redis client inline and export that from here instead.

Reply:
This has been simplified as part of this commit 8dc411e30f97b16e9ab4cd971ee62b1922422e1b